config/locales/en/providers.yml

```diff
@@ -1344,6 +1344,7 @@ en:
     check_capital_answers:
       show:
         h1-heading: Check your answers
+        inset_text: You cannot change the answers on this page once you save and continue.
```

Comment:
not a big deal but do we need this, since we are now calling `t("generic.cya_inset_text")` directly in the views. 

Apologies it it was not clear but what I was intending was something like 

```yml
   # anchor somewhere like generic.yml
   cya_inset_text: &cya_inset_text You cannot change the answers on this page once you save and continue.

   # splat/use in each view as relative translation
   inset_text: *cya_inset_text
```

Reply:
No, it looks like it got missed in the amendment tidy up. I'll remove and merge this :+1: